packages/components/src/snackbar/style.scss

```diff
@@ -29,13 +29,14 @@
 	}
 
 	.components-snackbar__content-with-icon {
-		margin-left: $grid-unit-30;
+		position: relative;
+		padding-left: $grid-unit-30;
```

Comment:
I was about to suggest using logical properties (inline-start instead of left), but I guess that it doesn't matter much in Sass files, since RTL styles should be automatically generated.